### PR TITLE
POC JQuery 3 support in CMS4

### PR DESCRIPTION
**Pull request by maxime-rainville**
A very dirty way to get JQuery 3 working in CMS 4

## Parent issue
- https://github.com/silverstripe/silverstripe-admin/issues/1351

### CODE REVIEW

File: client/src/legacy/SelectionGroup.js [hunk 1]
```diff
@@ -4,10 +4,9 @@ import $ from 'jquery';
 // require('../styles/legacy/SelectionGroup.scss');
 
 
-$(document).ready(function() {
+$(function() {
   // Support both bootstrap / legacy selectors
-	$('ul.SelectionGroup input.selector, ul.selection-group input.selector')
-    .live('click', function() {
+	$('ul.SelectionGroup, ul.selection-group').on('click', 'input.selector', function() {
```
Comment (maxime-rainville) [inline client/src/legacy/SelectionGroup.js:9]: `live` method doesn't exist anymore

File: thirdparty/jquery-entwine/dist/jquery.entwine-dist.js [hunk 2]
```diff
@@ -749,12 +749,8 @@ Sizzle is good for finding elements for a selector, but not so good for telling
 
 (function($){	
 	
-	/**
-	 * Add focusin and focusout support to bind and live for browers other than IE. Designed to be usable in a delegated fashion (like $.live)
-	 * Copyright (c) 2007 Jörn Zaefferer
-	 */
 	if ($.support.focusinBubbles === undefined)  {
-		$.support.focusinBubbles = !!($.browser.msie);
+		$.support.focusinBubbles = true;
```
Comment (maxime-rainville) [inline thirdparty/jquery-entwine/dist/jquery.entwine-dist.js:753]: No longer care about IE support.

File: thirdparty/jstree/jquery.jstree.js [hunk 3]
```diff
@@ -439,7 +439,7 @@
 					.undelegate(".jstree")
 					.removeData("jstree_instance_id")
 					.find("[class^='jstree']")
-						.andSelf()
+						.addBack()
```
Comment (maxime-rainville) [inline thirdparty/jstree/jquery.jstree.js:442]: `andSelf` should be swapped for `addBack`

File: thirdparty/jstree/jquery.jstree.js [hunk 4]
```diff
@@ -581,10 +581,10 @@
 				obj = this._get_node(obj);
 				if(obj === -1) { return this.get_container().find("> ul > li:first-child"); }
 				if(!obj.length) { return false; }
-				if(strict) { return (obj.nextAll("li").size() > 0) ? obj.nextAll("li:eq(0)") : false; }
+				if(strict) { return (obj.nextAll("li").length > 0) ? obj.nextAll("li:eq(0)") : false; }
```
Comment (maxime-rainville) [inline thirdparty/jstree/jquery.jstree.js:584]: `size()` should be replace by `length`

**Comment by GuySartorelli** [comment]
Closing in favor of https://github.com/silverstripe/silverstripe-admin/pull/1375

**Pull request closed without merging**